### PR TITLE
Fixes #6 and #7

### LINKED ISSUE #6: Volume unit µl
see https://github.com/Open-Systems-Pharmacology/OSPSuite.Core/issues/441

### PR DESCRIPTION
**Pull request by msevestre**

### CODE REVIEW

File: OSPSuite.Dimensions.xml [hunk 1]
```diff
@@ -419,7 +419,8 @@
       <BaseRepresentation lengthExponent="-3" massExponent="0" timeExponent="0" electricCurrentExponent="0" temperatureExponent="0" amountExponent="0" luminousIntensityExponent="0" />
       <Units>
         <Unit name="1/l" factor="1" />
-        <Unit name="1/ml" factor="1000" />
+        <Unit name="1/ml" factor="1E3" />
+        <Unit name="1/µl" factor="1E6" />
```
Comment (msevestre) [inline OSPSuite.Dimensions.xml]: also added to inverse volume to be consistent (#6)

File: OSPSuite.Dimensions.xml [hunk 2]
```diff
@@ -632,6 +633,7 @@
       <Units>
         <Unit name="l" factor="1" />
         <Unit name="ml" factor="1E-3" />
+        <Unit name="µl" factor="1E-6" />
```
Comment (msevestre) [inline OSPSuite.Dimensions.xml]: Fix for #6

File: OSPSuite.Dimensions.xml [hunk 3]
```diff
@@ -727,13 +729,14 @@
         <Unit name="(kg*dm²)/min²" factor="1" />
       </Units>
     </Dimension>
-    <Dimension name="Katal" baseUnit="kat" defaultUnit="mol/min">
```
Comment (msevestre) [inline OSPSuite.Dimensions.xml]: This was all wrong here. Kat is mol/s but it was defined as mol/min
also our base unit should be umol/min

SO whomever implemented that was clearly out of his mind

**Comment by Yuri05** [comment]
should this PR still be merged?

**Comment by msevestre** [comment]
This one yes but not the other one! Just told you I wanted to change it lol

**Comment by Yuri05** [comment]
>This one yes but not the other one! Just told you I wanted to change it lol

:) Well, it's not wrong. Base units are not the SI units but the "dimension specific" ones.

**Pull request merged**